docs/production-deployment/cloud/export.mdx

```diff
@@ -38,6 +38,12 @@ The Export feature in Temporal Cloud provides the following benefits:
 
 For pricing information, see [Temporal Cloud Pricing](/cloud/pricing).
 
+:::note
+
+If you have enabled encryption for your Workflow History, your exported data will also be encrypted.
```

Comment:
Can we make sure the terminology is consistent across this page and https://docs.temporal.io/production-deployment/data-encryption? So maybe we should say "If you have enabled encryption for the data handled in your Workflow Execution, the exported data will also be encrypted"

Reply:
Adopted.